R/read_asc_Andor.R

```diff
@@ -49,11 +49,33 @@ read_asc_Andor <- function(file = stop("filename or connection needed"),
 
 hySpc.testthat::test(read_asc_Andor) <- function() {
   context("read_asc_Andor")
-  test_that("Andor Solis .asc text files", {
-    skip("Need to adapt to new package")
-    expect_known_hash(
-      read_asc_Andor("fileio/asc.Andor/ASCII-Andor-Solis.asc"),
-      "9ead937f51"
-    )
+
+  andor <- system.file("extdata/fileio/asc.Andor/", "ASCII-Andor-Solis.asc",package = "hySpc.read.txt")
+  spc <- read_asc_Andor(andor)
+
+  test_that("Andor Solis .asc filename",{
```

Comment:
Maybe better not "Andor Solis .asc filename" but "Correct labels for Andor Solis .asc"